meshtastic/__main__.py

```diff
@@ -9,16 +9,15 @@
 import sys
 import time
 
-import pyqrcode
+import pyqrcode # type: ignore[import-untyped]
 import yaml
 from google.protobuf.json_format import MessageToDict
-from pubsub import pub
+from pubsub import pub # type: ignore[import-untyped]
 
 import meshtastic.test
 import meshtastic.util
-from meshtastic import channel_pb2, config_pb2, portnums_pb2, remote_hardware
+from meshtastic import channel_pb2, config_pb2, portnums_pb2, remote_hardware, BROADCAST_ADDR
 from meshtastic.version import get_active_version
-from meshtastic.__init__ import BROADCAST_ADDR
```

Comment:
Removing this import from `meshtastic.__init__` was something mypy needed to be happy, and is done a few different places. Otherwise it would complain about the same file being imported both as `meshtastic` and as `meshtastic.__init__`. Plus importing from `__init__` is a little bit funky anyway, pythonically speaking.